<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgRemovedSongs</class>
 <widget class="QDialog" name="DlgRemovedSongs">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1020</width>
    <height>664</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SkauTan: Removed songs</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableWidget" name="twRemoved">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="horizontalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Date</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Filename</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Del</string>
      </property>
      <property name="toolTip">
       <string>Was the file deleted from the disk</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string comment="# duplicates then">DupThen</string>
      </property>
      <property name="toolTip">
       <string>Number of duplicates in DB at the time of removal</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string comment="# duplicates now">DupNow</string>
      </property>
      <property name="toolTip">
       <string>Number of duplicates currently in DB</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnExport">
       <property name="text">
        <string>&amp;Export...</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnClear">
       <property name="text">
        <string>Clear...</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="text">
        <string>&amp;Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
